server.js

```diff
@@ -0,0 +1,143 @@
+
+// Syncing our database and logging a message to the user upon success
+async function tableUpdate() {
+  await db.sequelize.sync({ force: true });
+
+  await db.User.create({
+    email: "testy@mctesterson.net",
+    password: "test123"
+  });
+
+  await db.Category.bulkCreate([
+    { categoryName: "Books", apiCategoryId: 10 },
+    { categoryName: "Film", apiCategoryId: 11 },
+    { categoryName: "Music", apiCategoryId: 12 },
+    { categoryName: "Television", apiCategoryId: 14 },
+    { categoryName: "Video Games", apiCategoryId: 15 },
+    { categoryName: "Anime and Manga", apiCategoryId: 31 },
+    { categoryName: "Cartoons", apiCategoryId: 32 }
+  ]);
+
+  const quizURL =
+    "https://opentdb.com/api.php?amount=10&category=15&difficulty=easy&type=multiple";
+
+  axios.get(quizURL).then(async results => {
+    const dataSet = results.data;
+
+    const cleanResults = dataSet.results.map(result => {
+      const cleanResult = result;
+      cleanResult.question = unescapeHtml(result.question);
+      cleanResult.correct_answer = unescapeHtml(result.correct_answer);
+      cleanResult.incorrect_answers = result.incorrect_answers.map(
+        tempAnswer => {
+          return unescapeHtml(tempAnswer);
+        }
+      );
+      return result;
+    });
+
+    for (i = 0; i < cleanResults.length; i++) {
+      await db.Question.create({ questionText: cleanResults[i].question }); //it gets mad at this
+
+      // async function createQuestions() {
+      //   await db.Question.create({ questionText: cleanResults[i].question });
+      // }
+      // // db.Question.create({ questionText: cleanResults[i].question });
+      // await createQuestions();
+
+      function qcid() {
+        const id = i + 1;
+        return id;
+      }
+
+      async function createCorrect() {
+        await db.Answer.create({
+          answerText: cleanResults[i].correct_answer,
+          isCorrect: true,
+          QuestionId: await qcid()
+        });
+      }
+
+      console.log("\x1b[35m%s\x1b[0m", cleanResults[i].question); //purple
+      console.log("\x1b[36m%s\x1b[0m", cleanResults[i].correct_answer); //cyan
+
+      for (z = 0; z < cleanResults[i].incorrect_answers.length; z++) {
+        console.log("\x1b[31m%s\x1b[0m", cleanResults[i].incorrect_answers[z]); //red
+
+        async function createWrong() {
+          await db.Answer.create({
+            answerText: cleanResults[i].incorrect_answers[z],
+            isCorrect: false,
+            QuestionId: await qcid()
+          });
+        }
+
+        createWrong();
+      }
+      await createCorrect();
+    }
+
+    function unescapeHtml(text) {
+      return text
+        .replace(/&amp;/g, "&")
+        .replace(/&lt;/g, "<")
+        .replace(/&gt;/g, ">")
+        .replace(/&quot;/g, '"')
+        .replace(/&#039;/g, "'")
+        .replace(/&eacute;/g, "é")
+        .replace(/&Uuml;/g, "Ü")
+        .replace(/&rsquo;/g, "’");
+    }
+  });
+}
+
+async function earsAreHearing() {
```

Comment:
while amusing, you should avoid function names like this in favor of more descriptive function names